spec/script/setup_concepts_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'spec_helper'
+require 'rails_helper'
+require 'setup_concepts'
+
+VCR.configure do |config|
+  config.cassette_library_dir = "fixtures/vcr_cassettes"
+  config.hook_into :webmock # or :fakeweb
+end
+
+describe 'Setting up concepts' do
+  VCR.use_cassette('fetching concepts') do
+
+    setup = Setup::Concepts.new
+    
+    it "should have an attribute concepts" do
+      expect(setup.concepts.class).to eq(Array)
+    end
+
+    response = setup.fetch_concepts()
```

Comment:
@ddmck might be nice to have this in a before_each loop to make it explicit, I at first didn't notice this line